CSS/packages.css

```diff
@@ -0,0 +1,51 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    color: #333;
+    background-color: #f9f9f9;
+}
+
+header {
```

Comment:
Header should be same as home page to make this consistent. link navigationbar.css and add the html content related to nav bar to packages.html